packages/client/src/client/components/Gap.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components"
```

Comment:
for later, not sure we need a Gap component - prefer MUI style of layout with gaps, and then customise with margins for edge cases